components/automate-ui/src/app/pages/team-details/team-details.component.ts

```diff
@@ -135,38 +138,39 @@ export class TeamDetailsComponent implements OnInit, OnDestroy {
         this.isMinorV1 = version === 'v1';
       });
 
-    this.store.select(assignableProjects)
-      .subscribe((assignable: ProjectsFilterOption[]) => {
-        // compiles a list of all available projects
+    combineLatest([
+      this.store.select(assignableProjects),
+      this.store.select(optionsLoadingStatus),
+      this.store.select(v1TeamFromRoute),
+      this.store.select(v2TeamFromRoute),
+      this.store.select(iamMajorVersion)
+    ]).pipe(
+      takeUntil(this.isDestroyed),
+      map(([assignable, pStatus, v1Team, v2Team, major]:
+        [ProjectsFilterOption[], EntityStatus, Team, Team, IAMMajorVersion]) => {
+        this.isMajorV1 = major === 'v1';
+        return [assignable, pStatus, (this.isMajorV1 ? v1Team : v2Team)];
+      }),
+      filter(([_assignable, pStatus, team]:
+        [ProjectsFilterOption[], EntityStatus, Team]) =>
+        pStatus !== EntityStatus.loading && !!team
+      ),
```

Comment:
[nit] I find this a little hard to read. I guess I'd put the types on the same line -- just a matter of preference, though 😄